### ISSUE TITLE
Cannot make it work with dotnetcore

### ISSUE THREAD

**Issue by AlexAtNet** [linvi/tweetinvi#615]
Platform: dotnet core 2.1.2, macos.

Steps to reproduce:

1) dotnet new console --language F# --name news.test
2) cd news.test/
3) dotnet add news.test.fsproj package TweetinviAPI
4) add "open Tweetinvi" to Program.fs
5) dotnet build

Expecting all ok. Actual result: 

    Program.fs(4,6): error FS0039: The namespace or module 'Tweetinvi' is not defined.

**Comment by jskii** [comment]
Isn't it a C# library?

**Comment by AlexAtNet** [comment]
Resolved here: https://stackoverflow.com/questions/47779729/dotnet-adds-package-and-reference-but-it-does-not-appear-in-build/47781843#47781843

Apparently, "dotnet add news.test.fsproj package TweetinviAPI" installs some strange version...

**Comment by linvi** [comment]
Yes unfortunately the donet does not always pick up the correct version. I will see if this can be improved in the Nuget Package.
